v-next/hardhat-node-test-reporter/src/reporter.ts

```diff
@@ -244,45 +244,15 @@ export function hardhatTestReporter(
           yield event.data.message;
           break;
         }
-        case "test:plan": {
-          // Do nothing
-          break;
-        }
-        case "test:enqueue": {
-          // Do nothing
-          break;
-        }
-        case "test:dequeue": {
-          // Do nothing
-          break;
-        }
-        case "test:watch:drained": {
-          // Do nothing
-          break;
-        }
-        case "test:complete": {
-          // Do nothing
-          break;
-        }
         case "test:coverage": {
           yield chalk.red("\nTest coverage not supported by this reporter\n");
           break;
         }
-        // @ts-expect-error -- A new event type was added in https://github.com/nodejs/node/pull/54851
-        // but @types/node doesn't know about it yet. Once it does, the dependency should be updated
-        // and this comment removed.
-        case "test:summary": {
-          // Do nothing
-          break;
-        }
-        /* eslint-disable-next-line @typescript-eslint/switch-exhaustiveness-check --
-        We have this extra check here becase we know the @types/node type is
-        unreliable */
         default: {
-          const _isNever: never = event;
-          void _isNever;
-
-          yield chalk.red(`Unsuported node:test event:`, event);
+          // NOTE: Do nothing; we explicitly opt-in to the events we want to
```

Comment:
Maybe we can log it with `debug`

Reply:
I think if we do that we might run into some ordering edge cases as the reporter doesn't really log anything at the moment; it's just a string generator that is later piped through to the stdout.

I do agree with the sentiment though, it could be nice to be able to enable extra logging of unknown events.

Reply:
Good point. Let's play safe.